src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -67,6 +67,7 @@
   private JCheckBox myShowBuildMethodGuides;
   private JCheckBox myShowMultipleChildrenGuides;
   private JCheckBox myShowBuildMethodsOnScrollbar;
+  private JCheckBox myShowClosingLabelsInCheckBox;
```

Comment:
nit: maybe rename to `myShowClosingLabels` to stay consistent with other checkbox names

Reply:
👍 good catch (I'd relied on the naming in the GUI editor)